gammapy/data/observations.py

```diff
@@ -263,12 +263,16 @@ def create(
             location=location,
         )
 
-        meta = ObservationMetaData(
-            deadtime_fraction=deadtime_fraction,
-            location=location,
+        time_info = TimeInfoMetaData(
```

Comment:
Should we not check here that time_stop>time_start, per safety for the whole pipeline?

Reply:
Well maybe. But currently we don't validate that GTIs are correctly ordered. This might be something to add.